### PR TITLE
Feature/image options

### PR DESCRIPTION
**Pull request by egrace479**
Added option to select number of images to display (#7 ).

### CODE REVIEW

**Review by thompsonmj — APPROVED** [review]
Great!

File: test_components/proto-img-only.py [hunk 1]
```diff
@@ -129,8 +129,7 @@ def get_filename(subspecies, view, sex, hybrid):
     df_sub = df_sub.loc[df_sub.Sex.isin(sex)]
     df_filtered = df_sub.loc[df_sub.hybrid_stat.isin(hybrid)]
     if len(df_filtered) > 0:
-        df_filtered.sample(1)
-        filename = df_filtered.Image_filename.astype('string').values[0]
+        filename = df_filtered.sample().Image_filename.astype('string').values[0]
```
Comment (johnbradley) [inline test_components/proto-img-only.py:132]: This looks to be the same function as in dashboard.py, but it didn't get updated for the `num_images` parameter. Is there a reason to have two copies of this code and diverge them like this. Is it be possible to keep this function in a central location and re-use it in both places?

Comment (egrace479) [reply test_components/proto-img-only.py:132]: I was leaving a minimum working example in the `test_components` thinking they may be useful for someone in the future. I could pull some functions to a central location to declutter `dashboard.py` and potentially update components.

Comment (johnbradley) [reply test_components/proto-img-only.py:132]: Is the `test_components` version still useful to you when working on this app? From a maintenance perspective I am really likely to miss that I need to update the `test_components` version as well. Typically I try to keep the code to just what is needed. If someone wants an earlier version they could always find it in the older commits.

Comment (egrace479) [reply test_components/proto-img-only.py:132]: The `proto-image-ony` file could be useful for testing out some other features that may be added with the images (such as color bars) and there's a file with a helper function, but the other components aren't likely to be modified from there. The other reasoning was if I changed it to a multi-page app. From that perspective, I could update `proto-image-ony.py` to match and just get the other two functions in a separate file.

Comment (johnbradley) [reply test_components/proto-img-only.py:132]: Perhaps we should just create an issue to look into code reducing duplication.

**Review by johnbradley — APPROVED** [review]
LGTM

**Pull request merged**